egs/iwslt22_ta/ST/pruned_transducer_stateless5/decode.py

```
@@ -0,0 +1,944 @@
+            batch_str = f"{batch_idx}/{num_batches}"
+
+            logging.info(f"batch {batch_str}, cuts processed until now is {num_cuts}")
+    return results
+
+
+def save_results(
+    params: AttributeDict,
+    test_set_name: str,
+    results_dict: Dict[str, List[Tuple[str, List[str], List[str]]]],
+):
+    test_set_wers = dict()
```

Ruff (0.12.2)

Local variable `test_set_wers` is assigned to but never used

Remove assignment to unused variable `test_set_wers`

(F841)